exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>It would be good to give the reader a bit more info on what some of the columns in the table above mean. How did you decide to choose "swp22372" out of all the possible options, for example? Was there information in the table that affected that choice? It might be particularly good to draw the reader's attention to the Product Type column. I'm not sure what else might be relevant - what do you look at when analyzing your choices? "Description?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>